modules/apps/frontend-js/frontend-js-components-sample-web/src/main/resources/META-INF/resources/js/TranslationManagerSampleReact.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayLayout from '@clayui/layout';
+import {TranslationAdminSelector} from 'frontend-js-components-web';
+import React from 'react';
+
+export default function TranslationManagerSampleReact({
```

Comment:
It might lead to think that **all** the samples are in that file but, right now, other samples (aui and liferay-ui) are outside of it, isn't it confusing?

Reply:
I don't think so, it's a matter of redundancy, the component is built in React but there's no need here to specify it in the name.

Jsp + js components is a common pattern un DXP so it can be understood on its own.

Also, if we'd need to specify that those are React Sampled we should do the same with those that live in the jsp (which is a bit of overengineering).

Reply:
👍